scene/gui/spin_box.cpp

```diff
@@ -202,7 +202,8 @@ void SpinBox::_line_edit_focus_enter() {
 
 void SpinBox::_line_edit_focus_exit() {
 	// Discontinue because the focus_exit was caused by left-clicking the arrows.
-	if (get_viewport()->gui_get_focus_owner() == get_line_edit()) {
+	Viewport *viewport = get_viewport();
```

Comment:
```suggestion
	const Viewport *viewport = get_viewport();
```

Reply:
In order do to this I also had to make `Viewport::gui_get_focus_owner()` be const.